sdk-api-src/content/webauthenticationcoremanagerinterop/nf-webauthenticationcoremanagerinterop-iwebauthenticationcoremanagerinterop-requesttokenforwindowasync.md

```diff
@@ -47,34 +48,45 @@ Asynchronously requests a token from a web account provider. If necessary, the u
 
 ### -param appWindow
 
-Type: [HWND](https://docs.microsoft.com/windows/desktop/winprog/windows-data-types)
-
-The app window.
+The window to be used as the parent for the window prompting the user for credentials,

```

Comment:
owner, not parent. owner/owned is the relationship used in this case, parent child is different.